DigitalLearningSolutions.Web/Views/Frameworks/Developer/_SignpostingPaginationNav.cshtml

```diff
@@ -0,0 +1,45 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Frameworks
+@model CompetencyResourceSignpostingViewModel
+
+<nav class="nhsuk-pagination" role="navigation" aria-label="Pagination">
+  <form method="post" role="search" asp-controller="Frameworks" asp-action="GoToPage">
+    <div class="nhsuk-pagination__list">
+      <div class="pagination-button-container">
+        <div class="nhsuk-pagination-item--previous" @(Model.Page == 1 ? "hidden" : "")>
+          <button class="nhsuk-pagination__link nhsuk-pagination__link--prev" type="submit"
+                  asp-action="GoToPreviousPage"
+                  asp-controller="Frameworks">
+            <svg class="nhsuk-icon nhsuk-icon__arrow-left" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+              <path d="M4.1 12.3l2.7 3c.2.2.5.2.7 0 .1-.1.1-.2.1-.3v-2h11c.6 0 1-.4 1-1s-.4-1-1-1h-11V9c0-.2-.1-.4-.3-.5h-.2c-.1 0-.3.1-.4.2l-2.7 3c0 .2 0 .4.1.6z"></path>
+            </svg>
+            <span class="nhsuk-pagination__title">Previous</span>
+          </button>
+        </div>
+      </div>
+      <div class="page-indicator-container">
+        <span aria-live="polite" aria-atomic="true">
+          <span class="nhsuk-u-visually-hidden">page </span>
+          <span id="page-indicator">@Model.Page of @Model.TotalPages</span>
+        </span>
+      </div>
+      <div class="pagination-button-container">
+        <div class="nhsuk-pagination-item--next" @(Model.Page < Model.TotalPages ? "" : "hidden")>
+          <button class="nhsuk-pagination__link nhsuk-pagination__link--next" type="submit"
```

Comment:
This button should also be converted to a link to call a Get method and should have search string and page number as route values, e.g.:
```
asp-route-search="@Model.SearchString"
asp-route-page="@(Model.Page+1)"
```

Reply:
Change done and styles updated.